src/utils.cpp

```diff
@@ -284,3 +289,12 @@ bool Utils::IsBigEndian() {
 
     return(d.c[0] == 1);
 }
+
+int32_t Utils::GetRandomNumber(int32_t from, int32_t to) {
+	std::uniform_int_distribution<int32_t> dist(from, to);
```

Comment:
Using int32_t as the numeric type might be undefined behavior. From the [documentation](http://en.cppreference.com/w/cpp/numeric/random/uniform_int_distribution):
>The effect is undefined if this is not one of short, int, long, long long, unsigned short, unsigned int, unsigned long, or unsigned long long. 

See also [this](https://groups.google.com/a/isocpp.org/forum/#!msg/std-discussion/rOzm69RSNcg/aIqYf__XJb0J)